src/App.cpp

```diff
@@ -573,6 +575,59 @@ void createAGVs()
                 }
                 socialForce->addAGV(agv);
 
+                int marker = numRunPerHallway * (juncIndexTemp + 1) - 1;

```

Comment:
I believe the program itself in runMode == 1 doesn't require to change anything. Am I missing something here?

Reply:
Ah never mind, it's the exact same thing in the runMode == 3. Nothing has been changed.